src/Momento.Sdk/Momento.Sdk.csproj

```diff
@@ -32,25 +32,22 @@
 		<PackageProjectUrl>https://github.com/momentohq/client-sdk-dotnet</PackageProjectUrl>
 		<RepositoryUrl>https://github.com/momentohq/client-sdk-dotnet</RepositoryUrl>
 	</PropertyGroup>
-	<ItemGroup>
-		<PackageReference Include="Google.Protobuf" Version="3.19.0" />
-		<PackageReference Include="Grpc.Net.Client" Version="2.40.0" />
-		<PackageReference Include="Grpc.Core" Version="2.41.1" />
-		<PackageReference Include="Microsoft.Extensions.Logging" Version="6.0.0" />
-		<PackageReference Include="Microsoft.SourceLink.Github" Version="1.1.1">
-		  <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
-		  <PrivateAssets>all</PrivateAssets>
-		</PackageReference>
```

Comment:
I did all of this through the Visual Studio UI.  I started by removing all deps, then added them back in until it compiled and tests pass.  I suspect that this Sourcelink.Github thing is something we want to keep but the build and tests all pass without it and I'm not completely sure what it is, so wanted to ask.  I assume I should add it back.

Reply:
@malandis this is what you just added to get the source code bundled with the SDK, yes?

Reply:
@pgautier404 @cprice404 Yes I added this today to get the source code linked. It's only used at build time: https://github.com/dotnet/sourcelink

Reply:
@malandis I pushed a commit that adds that back, along with the logging lib.